server/channels/app/admin.go

```diff
@@ -111,23 +111,23 @@ func AddLocalLogs(logData map[string][]string, s *Server, page, perPage int, ser
 	return nil
 }
 
-func (a *App) QueryLogs(c request.CTX, page, perPage int, logFilter *model.LogFilter) (map[string][]string, *model.AppError) {
-	return a.Srv().QueryLogs(c, page, perPage, logFilter)
+func (a *App) QueryLogs(rctx request.CTX, page, perPage int, logFilter *model.LogFilter) (map[string][]string, *model.AppError) {
+	return a.Srv().QueryLogs(rctx, page, perPage, logFilter)
 }
 
-func (a *App) GetLogs(c request.CTX, page, perPage int) ([]string, *model.AppError) {
-	return a.Srv().GetLogs(c, page, perPage)
+func (a *App) GetLogs(rctx request.CTX, page, perPage int) ([]string, *model.AppError) {
+	return a.Srv().GetLogs(rctx, page, perPage)
 }
 
 func (s *Server) GetLogsSkipSend(page, perPage int, logFilter *model.LogFilter) ([]string, *model.AppError) {
 	return s.platform.GetLogsSkipSend(page, perPage, logFilter)
 }
 
-func (a *App) GetLogsSkipSend(page, perPage int, logFilter *model.LogFilter) ([]string, *model.AppError) {
+func (a *App) GetLogsSkipSend(rctx request.CTX, page, perPage int, logFilter *model.LogFilter) ([]string, *model.AppError) {
```

Comment:
Let's blank these to make it clear that they are unused
```suggestion
func (a *App) GetLogsSkipSend(_ request.CTX, page, perPage int, logFilter *model.LogFilter) ([]string, *model.AppError) {
```

Reply:
Can you please apply the same change to all method where `request.CTX` is unused?